server/server/server.js

```diff
@@ -0,0 +1,27 @@
+const prisma = require('./db/db')

```

Comment:
This task is only to set up prisma and the users table so there shouldn't need to be any additions of code in the server.js file